GitlabCI.md

```diff
@@ -0,0 +1,83 @@
+Gitlab CI
+---
+
+The Duke Dataservice is making use of GitlabCI to augment the standard
+build and deployment process currently provided  by CircleCI. The
+GitlabCI platform allows us to test different processes to monitor the
+quality of our code, such as linting and code coverage, that do easily
+not fit into the CircleCI structure.
+
+#### Gitlab Github Mirroring
+For this to work, a project must be created in a Gitlab host that
+supports GitHub mirroring (this is an Enterprise feature). The project
+must mirror the github owner/repo to a GitHub owner/repo. **NOTE**
```

Comment:
Should one of these `github`s be a `gitlab`?

Reply:
good catch.